### PR TITLE
commit for multi projects and reimporting existing workflows with sam…

### PR DESCRIPTION
**Pull request by stikkireddy**
…e name

### CODE REVIEW

**Comment by asingamaneni** [comment]
Tested and ran successfully!

image

File: brickflow/engine/compute.py [hunk 1]
```diff
@@ -84,6 +84,7 @@ class Cluster:
     enable_elastic_disk: Optional[bool] = None
     driver_instance_pool_id: Optional[str] = None
     instance_pool_id: Optional[str] = None
+    runtime_engine: Optional[Literal["STANDARD", "PHOTON"]] = None
```
[the remark this answers is not in the document]
Comment (stikkireddy) [reply brickflow/engine/compute.py:87]: by default null is default where server defaults to standard. following the api spec on this one. 

runtime_engine
string
Enum: "NULL" "STANDARD" "PHOTON"
Decides which runtime engine to be use, e.g. Standard vs. Photon. If unspecified, the runtime engine is inferred from spark_version.

NULL is the first value. Optional is the indication for that.

[the remark this answers is not in the document]
Comment (asingamaneni) [reply brickflow/engine/compute.py:87]: Thanks for the clarification!

File: examples/brickflow_examples/workflows/resolver.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-from brickflow import RelativePathPackageResolver
+from brickflow.resolver import RelativePathPackageResolver
```
[the remark this answers is not in the document]
Comment (asingamaneni) [reply examples/brickflow_examples/workflows/resolver.py:2]: This is fixed by removing it

**Review by asingamaneni — APPROVED** [review]
Looks good to me and tested for all the changes. No issues found!

**Pull request merged**